mozilla/foundation.mozilla.org
# left aligned buttons to text content on article pages

Closes #7788 
Related PRs/issues #

**Link to test page:** foundation-s-article-bu-slc7zt.mofostaging.net/en/publication-page-with-child-article-pages/child-paper-else/

****Steps to test**:** 

1. Please visit the testing site above
2. Scroll to find the button saying "View the minimum election standards"
3. It should be left aligned with the rest of the text.
4. It should also wrap so all the text stays on the page throughout breakpoints
5. If it is behaving as expected, testing is complete!

### Percy Screenshots:
Showing the difference between running off-screen on mobile and now wrapping to show all content
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/18314510/140862002-89b6a8f7-e1b9-4ac2-8ddb-ccdc92c3a021.png">

Showing that there are no differences to buttons otherwise:
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/18314510/140862062-d033d8e1-824e-43fe-95a9-430fb130af25.png">
